pages/blog/async-api-spec-3.0-release.md

```diff
@@ -0,0 +1,59 @@
+-   [Move binding JSON schema files to main JSON schema repository](https://github.com/asyncapi/bindings/issues/113)
+
+We are using the [GitHub milestone](https://github.com/asyncapi/spec/milestone/18) to track down all the issues related to Spec 3.0 release. If you are willing to take up an issue, filter the issues with `needs champion` label and drive the initiative forward. We also have a [release journal](https://github.com/asyncapi/community/issues/163) for the upcoming Spec 3.0 release to keep note of all the references, build the right documentation, and track the progress for the upcoming release.
+
+## Contributing to the release
+
+AsyncAPI is a community-driven project and we welcome all kinds of contributions! To get started, join our [Slack workspace](http://asyncapi.com/slack-invite). After joining, reach out to us on the `#03_specification` channel and participate in the conversations.
+
+To better understand AsyncAPI and how the Spec 3.0 release will play out, we recommend watching our [past meeting recordings](https://www.youtube.com/watch?v=CLNgLB4-UnA&list=PLbi1gRlP7pihClJY-kXuTRRJ8n1awb0VV). To know how you can contribute, we recommend checking out our [contributing guidelines](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md). If you wish to keep track of all the work that is happening on the Spec 3.0 release, we recommend checking out our [GitHub issue](https://github.com/asyncapi/spec/issues/691).
+
+Before landing in your contributions, make sure you follow our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and the pull request you raised is [described properly](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits). To know how we collaborate using Git & GitHub, follow the [Git workflow](https://github.com/asyncapi/.github/blob/master/git-workflow.md) that we have developed for new contributors.
+
+## When will the release happen?
+
+We are yet to arrive at a date but the community is working towards releasing this by the end of 2022. Currently, we meet every two weeks to discuss the upcoming Spec 3.0 release and share updates and you are welcome to join it! We are going to have our next meeting on [UTC 16:00 on March 3, 2022](https://github.com/asyncapi/community/issues/270). You can access the [AsyncAPI Calendar](https://calendar.google.com/calendar/u/0/embed?src=tbrbfq4de5bcngt8okvev4lstk@group.calendar.google.com) and explore our upcoming meetings.
+
+The release would follow a three-step process: End of the review period, Beta testing and official announcement. Currently, we are working in a review period where important work is picked before we start working on them. During Beta testing, we invite external contributors and users to test the contributions that have landed in and validate their use cases. Finally, after rigorous testing, we will make an official announcement about our release!
```

Comment:
We don't really have any beta testing, do we? 🤔 

What exactly are you referring to here? 🙂

Reply:
I took some inspiration from this blog: https://www.asyncapi.com/blog/v2-important-dates#:~:text=the%20next%20version.-,Beta%20testing%20program%20(July%2DSeptember),-Right%20after%20we%27re

Let me know if that's not the norm right now and I will remove this portion 😄

Reply:
As we don't have any official way of doing this yet, maybe this makes more sense?
```suggestion
The release will happen gradually, where the new specification and tooling will live in coexistence until there is a collective consensus that everything is ready. This gives everyone an easy way to test out the new changes and provide feedback before the release. Finally, after rigorous testing, we will make an official announcement about our release!
```

Reply:
@derberg what do you think about this change? 🤔

Reply:
not sure what that means 😅 you mean release candidates?

Reply:
Yes 🙂

Reply:
That makes sense 👍 @HarshCasper can you adapt the sentenance? 🙂